File: Programming Basics with JAVA/4. Complex Conditional Statements/src/DayOfWeek.java
import java.util.Locale;
import java.util.Scanner;

public class DayOfWeek {

    public static void main(String... args) {
        Locale.setDefault(Locale.ROOT);
        Scanner input = new Scanner(System.in);
        switch (input.nextInt()){
            case 1:
                System.out.println("Monday");
                break;
            case 2:
                System.out.println("Tuesday");
                break;
            case 3:
                System.out.println("Wednesday");
                break;
            case 4:
                System.out.println("Thursday");
                break;
            case 5:
                System.out.println("Friday");
                break;
            case 6:
                System.out.println("Saturday");
                break;
            case 7:
                System.out.println("Sunday");
                break;
            default:
                System.out.println("Error");
        }
    }
}

File: Tech Module/Programming Fundamentals/13. Arrays/Arrays - Lab/01.DayOfWeek/DayOfWeek.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _01.DayOfWeek
{
    class DayOfWeek
    {
        static string[] dayOfWeek = new string[] { "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" };
        static void Main(string[] args)
        {
            int dayNumber = int.Parse(Console.ReadLine());
            try
            {
                Console.WriteLine(dayOfWeek[dayNumber - 1]);
            }
            catch (IndexOutOfRangeException)
            {
                Console.WriteLine("Invalid Day!");
            }
        }
    }
}
